compute/etc/sql_exporter/queries/autoscaling/lfc_approximate_working_set_windows.sql

```diff
@@ -0,0 +1,4 @@
+SELECT
```

Comment:
Why is this the only file in `autoscaling/` ?

Reply:
That's the only query that is different from the compute collector.